src/pages/workspace/AccessOrNotFoundWrapper.tsx

```diff
@@ -175,4 +185,7 @@ export default withOnyx<AccessOrNotFoundWrapperProps, AccessOrNotFoundWrapperOny
     isLoadingReportData: {
         key: ONYXKEYS.IS_LOADING_REPORT_DATA,
     },
+    betas: {
+        key: ONYXKEYS.BETAS,
+    },
```

Comment:
why do we need this change?

Reply:
It's used in `ReportUtils.canCreateRequest()`, but I will change to use `allBetas` from ReportUtils instead of passing to the function 👍